.dockerignore

```diff
@@ -1,2 +1 @@
-.git
```

Comment:
Could we still ignore the `.git/modules` and `.git/objects` sub-dirs?

Reply:
No. I tried ignoring each of them, but probably git does not work if something is missing. Built actually uses the installed git to grab git info